pkg/virt-controller/watch/vmi.go

```diff
@@ -1282,9 +1300,22 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 		}
 
 		if vmiSpecIfaces, vmiSpecNets, dynamicIfacesExist := network.CalculateInterfacesAndNetworksForMultusAnnotationUpdate(vmi); dynamicIfacesExist {
-			if err := c.updateMultusAnnotation(vmi.Namespace, vmiSpecIfaces, vmiSpecNets, pod); err != nil {
+			errMessage := fmt.Errorf("failed to hot{un}plug network interfaces for vmi [%s/%s]", vmi.GetNamespace(), vmi.GetName())
+
+			var networkToIPAMClaimParams map[string]network.IPAMClaimParams
+			if c.clusterConfig.PersistentIPsEnabled() {
+				var err error
+				networkToIPAMClaimParams, err = c.ipamClaimsManager.GetNetworkToIPAMClaimParams(vmi.Namespace, vmi.Name, vmiSpecNets)
+				if err != nil {
+					return &syncErrorImpl{
+						err:    fmt.Errorf("%s: %w", errMessage, err),
+						reason: FailedHotplugSyncReason,
```

Comment:
The error seems wrong.

Reply:
It failed hot{un}plug flow (*) due to failure of GetNetworkToIPAMClaimParams
and wrapping the error of `GetNetworkToIPAMClaimParams`
wdyt? or you prefer something that will not mention hotplug (less good imho because we are on "hotplug" segment)

(*) beside this [case](https://github.com/kubevirt/kubevirt/pull/11410#discussion_r1584412586) which isn't hot plug